ocpmodels/trainers/base_trainer.py

```diff
@@ -131,13 +132,14 @@ def __init__(
         else:
             self.config["dataset"] = dataset
 
-        if not is_debug and distutils.is_master():
+        if not is_debug and distutils.is_master() and not is_hpo:
```

Comment:
Is there an instance where `is_hpo` and `is_debug` are not used identically for these checks? I'm thinking we just add this at the top of the trainers and we can remove all these additional checks.
```
if is_hpo:
   is_debug = True
```

Reply:
Even if right now they're identical, might be better to keep them explicit and separate. We might not remember to make sure the HPO flow doesn't break every time we add something under `is_debug`.

Reply:
Going to leave this as is for the moment